src/common/stores/emote-menu-view-store.js

```diff
@@ -169,6 +193,27 @@ class EmoteMenuViewStore extends SafeEventEmitter {
         emotes.getEmotesByCategories([EmoteCategories.FRANKERFACEZ_GLOBAL])
       ),
     ];
+
+    for (const [registeredCategoryId, registeredCategory] of Object.entries(registeredProviderCategories)) {
+      if (registeredCategory?.channelId != null && registeredCategory?.channelId !== currentChannel?.id) {
+        continue;
+      }
+      const provider = this.getRegisteredProvider(registeredCategory.providerId);
+      if (provider == null) {
+        continue;
+      }
+      const category = createCategory(
+        registeredCategoryId,
+        registeredCategory.providerId,
+        `${provider.displayName} ${registeredCategory.channelId == null ? 'Global' : 'Channel'}`,
+        registeredCategory.channelId == null
+          ? Icons.IMAGE(provider.iconSrc, registeredCategoryId)
```

Comment:
instead of using an id as the alt text, why not use its name?

```
const categoryName = `${provider.displayName} ${registeredCategory.channelId == null ? 'Global' : 'Channel'}`;
```

this also doesn't support i18n though, so that's a bit of a pickle.